kratos/tests/cpp_tests/processes/test_calculate_discontinuous_distance_to_skin_process.cpp

```diff
@@ -364,10 +364,10 @@ namespace Testing {
 
         // Check elemental distance values
         const auto &r_elem_dist = volume_part.ElementsBegin()->GetValue(ELEMENTAL_DISTANCES);
-        KRATOS_CHECK_NEAR(r_elem_dist[0], 1.22382, 1e-5);
-        KRATOS_CHECK_NEAR(r_elem_dist[1], 1.22382, 1e-5);
-        KRATOS_CHECK_NEAR(r_elem_dist[2], 1.22382, 1e-5);
-        KRATOS_CHECK_NEAR(r_elem_dist[3], 1.22382, 1e-5);
+        KRATOS_CHECK_NEAR(r_elem_dist[0], 0.135661, 1e-5);
+        KRATOS_CHECK_NEAR(r_elem_dist[1], 0.135661, 1e-5);
+        KRATOS_CHECK_NEAR(r_elem_dist[2], 0.135661, 1e-5);
+        KRATOS_CHECK_NEAR(r_elem_dist[3], 0.135661, 1e-5);
```

Comment:
Mentioning that these changes are due to the fact that we were not properly computing the domain characteristic length (we were not correctly initializing the min/max values). In this test, as the volume domain is not centered in the (0,0,0) the characteristic length was wrong. Since this is used to initialize the values in the non intersected case, as the one we are testing in here, the values need to be updated accordingly.